main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+// All the questions that the program asks the user
+    noun = input("Choose a noun: ")
+    plural_noun = input("Choose a plural noun: ")
+    second_noun = input("Choose a noun: ")
+    place = input("Name a place: ")
+    adjective = input("Choose an adjective (Describing word): ")
+    third_noun = input("Choose a noun: ")
+// Displays the story based on the users input
+    print ("------------------------------------------")
+    print ("Be kind to your",noun,"- footed", plural_noun)
+    print ("For a duck may be somebody's", seond_noun,",")
```

Comment:
There are a few typos here. This won't work, since `seond_noun` is not ever defined. 
Also, `","` seems to be a typo. It should be a `"."`, as it seems like a complete sentence like the other `print()` statements.

I also wanted to point out that using the `,` within `print()` will allow you to combine strings separated with spaces. If you want to prevent those spaces, you may use `+` to concatenate strings or f-strings.
```suggestion
    print("For a duck may be somebody's " + second_noun + ".")
```
Another option with f-strings: 
```suggestion
    print(f"For a duck may be somebody's {second_noun}.")
```